### PR TITLE
Saucelabs

### PR DESCRIPTION
**Pull request by hukka**

### CODE REVIEW

File: fixtures/common.py [hunk 1]
```diff
@@ -4,15 +4,37 @@
...
-        chrome_options.add_argument('--disable-gpu')
-    chrome_options.add_argument('--window-size=512,512')
-    driver = webdriver.Chrome(chrome_options=chrome_options)
+    if settings.SAUCELABS_USERNAME:
+        if settings.BROWSER == 'ie11':
+            desired_cap = {
+                'platform': "Windows 10",
+                'browserName': "internet explorer",
+                'version': "11.103",
+                }
```
Comment (juyrjola) [inline fixtures/common.py]: The closing braces should be indented one level left.

File: fixtures/common.py [hunk 2]
```diff
@@ -4,15 +4,37 @@
...
+                'version': "10.0",
+                }
+        else:
+            desired_cap = {
+                    'platform': "Mac OS X 10.9",
+                    'browserName': "chrome",
+                    'version': "31",
+                    }
+        driver = webdriver.Remote(
+           command_executor=f'http://{settings.SAUCELABS_USERNAME}:{settings.SAUCELABS_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub',
```
Comment (juyrjola) [inline fixtures/common.py]: Shouldn't it be `https`?

**Review by juyrjola — CHANGES_REQUESTED** [review]

File: fixtures/common.py (hunk 2 above)
Comment (hukka) [reply fixtures/common.py]: Unfortunately no… Using https just hangs the test, and the documentation has plain http links too.

File: fixtures/common.py (hunk 1 above)
Comment (hukka) [reply fixtures/common.py]: Ok, but we need to document that.

**Pull request merged**